packages/nova-react-test-utils/README.md

```diff
@@ -277,7 +277,60 @@ const meta = {
 } satisfies Meta<typeof FeedbackContainer>;
 ```
 
-The second parameter of `getNovaDecorator` is an `options` object of type `Partial<EnvironmentConfig>` from `relay-test-utils`.
+The second parameter of `getNovaDecorator` is an `getEnvironmentOptions` callback that should return object of type `Partial<EnvironmentConfig>` from `relay-test-utils`.
+
+#### Can I swap out the graphitation MockPayloadGenerator for something else?
+
+Yes, in `getNovaDecorator` in `options` param you can supply a `generateFunction` which will be used to
+generate data instead of the `MockPayloadGenerator`. Use this if you want to use the
+`MockPayloadGenerator` supplied by `relay-test-utils`.
+
+#### (Pure relay or Nova with Relay) How can I make sure the mock data is generated for client extensions?
+
+The current default generator doesn't support client extensions. However, you can use the `generateFunction` option to provide your own generator. Here is an example of how you can use the `MockPayloadGenerator` from `relay-test-utils` to generate data for client extensions:
+
+```tsx
+import { MockPayloadGenerator } from "relay-test-utils";
+import { getNovaDecorator } from "@nova/react-test-utils/relay";
+
+const novaDecorator = getNovaDecorator(schema, {
+  generateFunction: (operation, mockResolvers) => {
+    const result = MockPayloadGenerator.generate(
+      operation,
+      mockResolvers ?? null,
+      {
+        mockClientData: true, // this makes sure data for client extensions is generated
+      },
+    );
+
+    return result;
+  },
+});
+```
+
+#### (Pure relay or Nova with Relay) How can I make sure the mock data is generated with deferred payloads?
+
+The current default generator doesn't return deferred payloads. However, similarly as in example above you can use the `generateFunction` option to provide your own generator which does.
+
+```tsx
+import { MockPayloadGenerator } from "relay-test-utils";
+import { getNovaDecorator } from "@nova/react-test-utils/relay";
+
+const novaDecorator = getNovaDecorator(schema, {
+  generateFunction: (operation, mockResolvers) => {
+    const result = MockPayloadGenerator.generateWithDefer(
+      operation,
+      mockResolvers ?? null,
+      {
+        mockClientData: true,
+        generateDeferredPayload: true, // this makes sure mock data is array of deferred payloads, not a single response
```

Comment:
Wouldn't people want this for most use cases? I'm thinking it would be nice to make this the default behavior.

Could we expose a default `generateFunction` from `@nova/react-test-utils/relay` that does exactly this?

Reply:
> Even if you check test cases in relay-test-utils a lot of snapshots just don't make sense

Probably our brother GPT that made them :upside_down_face: 

Yes, let's not do this yet.